src/test/resources/uk/gov/laa/gpfd/health_api.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Call Actuator Endpoint
+
+  Scenario: Return 200 response
+    Given the service is running and we are not logged in
+    When it calls the actuator endpoint
+    Then it should return a 200 response
+
+  Scenario: Return 201 response
```

Comment:
**Suggestion**: Double-check the scenario. It appears to expect a 201 response but verifies against 200.

Reply:
That was a second test to check the BeforeAll stuff was working properly, should have been deleted 🙈 Nice spot